tests/test_retrievers_langchain_vdms.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -xe
+
+WORKPATH=$(dirname "$PWD")
+LOG_PATH="$WORKPATH/tests"
+ip_address=$(hostname -I | awk '{print $1}')
+no_proxy=$no_proxy,$ip_address
+
+function build_docker_images() {
+    cd $WORKPATH
+    hf_token="dummy"
+    docker build --no-cache -t opea/retriever-vdms:comps \
+        --build-arg https_proxy=$https_proxy \
+        --build-arg http_proxy=$http_proxy \
+        --build-arg huggingfacehub_api_token=$hf_token\
+        -f comps/retrievers/langchain/vdms/docker/Dockerfile .
+
+}
+
+function start_service() {
+    #unset http_proxy
+    # vdms
+    vdms_port=55555
+    docker run -d --name test-comps-retriever-vdms-vector-db \
+        -p $vdms_port:$vdms_port   intellabs/vdms:latest
+    sleep 10s
+
+    # tei endpoint
+    tei_endpoint=5008
+    model="BAAI/bge-base-en-v1.5"
+    docker run -d --name="test-comps-retriever-tei-endpoint" \
+        -p $tei_endpoint:80 -v ./data:/data \
+        -e HTTPS_PROXY=$https_proxy -e HTTP_PROXY=$https_proxy \
+        --pull always ghcr.io/huggingface/text-embeddings-inference:cpu-1.2 \
+        --model-id $model
+    sleep 30s
+
+    export TEI_EMBEDDING_ENDPOINT="http://${ip_address}:${tei_endpoint}"
+
+    export INDEX_NAME="rag-vdms"
```

Comment:
Same for here and line 50 to use COLLECTION_NAME

Reply:
Thanks for pointing this out. Changed INDEX_NAME to COLLECTION_NAME.